src/config/app.config.module.ts

```diff
@@ -1,18 +1,21 @@
 import { Module } from '@nestjs/common';
 import { TypeOrmModule } from '@nestjs/typeorm';
-import databaseConfig from './ormconfig';
-import { ServerHealthCheckModule } from 'src/modules/server-health-check/server-health-check.module';
 import { RouterModule } from '@nestjs/core';
 
-import { NestConfigModule } from './nest.config.module';
-import { AuthModule } from 'src/modules/auth/auth.module';
+import databaseConfig from '@config/ormconfig';
+import { NestConfigModule } from '@config/nest.config.module';
+
+import { ServerHealthCheckModule } from '@server-health-check/server-health-check.module';
+import { AuthModule } from '@auth/auth.module';
+import { FeeChallanModule } from '@fee-challan/fee-challan.module';
 
 @Module({
   imports: [
     NestConfigModule,
     TypeOrmModule.forRoot(databaseConfig),
     ServerHealthCheckModule,
     AuthModule,
+    FeeChallanModule,
```

Comment:
Should this not be renamed to Result-Workflow or something like that?

Reply:
BTW, I believe we dont even need this module now, as the requests for the contract will be forwarded from API Gateway, right?
So this should only be calling the contract functions based on the request received?

Reply:
With reference to [this](https://github.com/CyFy-Blockchain/fabric-gateway/pull/20#issuecomment-2508733131) message, I think API Gateway should not know anything about contract calling. It should be just another API for it. Fabric GW should be responsible for how it maps the incoming request to the contract call.